src/index.ts

```diff
@@ -294,10 +294,10 @@ class BuxClient implements TransportService {
    * Admin only: Delete a paymail
    *
    * @param address string Paymail address (ie. example@bux.org)
-   * @return {PaymailAddress}
+   * @return void
    */
-  async AdminDeletePaymail(address: string): Promise<PaymailAddress> {
-    return await this.client.httpTransport.AdminDeletePaymail(address);
+  async AdminDeletePaymail(address: string) {
```

Comment:
Shouldn't we sent some confirmation that paymail has been successfully deleted ?

Reply:
if there is no error, paymail has been deleted